apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.py

```diff
@@ -0,0 +1,133 @@
+import base64
+import json
+from pathlib import Path
+
+import chevron
+import lxml.html
+import prairielearn as pl
+from lxml.html import HtmlElement
+from typing_extensions import assert_never
+
+ATTR_ANSWER_NAME = "answers-name"
+ATTR_WIDTH = "width"
+ATTR_HEIGHT = "height"
+ATTR_SOURCE_FILE_NAME = "source-file-name"
+ATTR_SOURCE_DIRECTORY = "directory"
+
+
+SOURCE_DIRECTORY_MAP = {
+    "serverFilesCourse": "server_files_course_path",
+    "clientFilesCourse": "client_files_course_path",
+    "clientFilesQuestion": "client_files_question_path",
+    "courseExtensions": "course_extensions_path",
```

Comment:
What's the rationale for supporting this directory in this element?

Reply:
I don't have one :) 

I picked the list from `pl-template.py`.

Reply:
IMO we should drop this and only add it if someone specifically requests it. I can't currently see a good reason to support this.